api/wikis/serializers.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer, IDField, TypeField, Link, LinksField, RelationshipField
+from api.base.utils import absolute_reverse
+
+
+class WikiSerializer(JSONAPISerializer):
+
+    filterable_fields = frozenset([
+        'name',
+        'date_modified'
+    ])
+
+    id = IDField(source='_id', read_only=True)
+    type = TypeField()
+    name = ser.CharField(source='page_name')
+    path = ser.SerializerMethodField()
+    materialized = ser.SerializerMethodField(method_name='get_path')
+    date_modified = ser.DateTimeField(source='date')
```

Comment:
I believe this should just be `modified` (see files serializer in case the docs that I copied this from were wrong)